dashboard/src/utils/dateFormatter.js

```diff
@@ -1,3 +1,7 @@
+import { isValidDate } from "@patternfly/react-core"
+
 export const formatDate=(date)=>{
-    return date.toISOString().slice(0,10)
+    if (!isValidDate(date)) return;
+    const dateWithoutOffset=date.toString().split(/\+|-/)
+    return (new Date(dateWithoutOffset)).toISOString().split('T')[0]
 }
```

Comment:
My reading of this is that this function receives a `Date` object -- I'm guessing that it represents a date in local time; we're creating a string with the timezone offset removed; we're then creating a `Date` using the butchered time string as though it were local time; and then we're returning the string for that date with the time portion removed.  What are we _trying_ to accomplish with all of these gymnastics?  (A docstring or other code comment would be _really_ helpful!!)

What I would like to propose is that all date-times should be in UTC.  Thus, the `date` value, should properly represent the date and time that should be formatted.  If that's the case, then `toISOString()` will do the right thing (reporting the value in UTC time), and you can trim off the `T`, the time, and the trailing `Z` in one fell swoop if all you want to display is the date.

That is, this function should reduce to just:
```
export const formatDate=(date)=>{
    if (isValidDate(date))
      return date.toISOString().split('T')[0]
}
```
and, it might not even need the `isValidDate()` test....